pkg/ui/workspaces/cluster-ui/src/insights/types.ts

```diff
@@ -64,6 +64,15 @@ export type TransactionInsightEventDetails = {
   execType: InsightExecEnum;
 };
 
+export type InsightContentionEvent = {
```

Comment:
Can we clarify this is for stmts right now? Since we have a `BlockedContentionDetails` for transactions already it can get a little confusing

Reply:
Ideally we'd be able to merge the 2 in the future but I'm fine with keeping it separate right now since the other is based off the contention registry response